src/maps/components/StandardMap.vue

```diff
@@ -245,12 +257,38 @@ export default {
       if (this.bounds) {
         // unfortunately it doesn't auto update when we change the padding, so we have to do it ourselves...
         // the $nextTick improved the behaviour when resizing in practise
-        this.$nextTick(() => this.$refs.map.fitBounds(this.bounds))
+        this.$nextTick(() => {
+          if (!this.leafletObject) return
+          this.leafletObject.fitBounds(this.bounds)
+        })
       }
     },
+    /* TODO use or drop
+    bounds (bounds) {
+      this.$nextTick(() => {
+        if (!this.leafletObject) return
+        if (!bounds) return
+        if (!bounds.isValid()) return
+        this.leafletObject.flyToBounds(bounds)
+      })
+    },
+    centerAndZoom ([center, zoom]) {
+      this.$nextTick(() => {
+        if (!this.leafletObject) return
+        if (!center) return
+        if (!zoom) return
+        this.leafletObject.flyTo(center, zoom)
+      })
+    },
+    */
```

Comment:
I guess a bit more work here? Or can have an issue and deal with it later... given the other map niggles?

Reply:
this code actually worked quite nicely, but it made debugging the other map problems even harder, so I'm inclined to leave it out.

....overall, maybe we want to have the whole sidenav map less prominent?! some flyto animation would counter this.